src/main/frontend/handler/file_sync.cljs

```diff
@@ -0,0 +1,93 @@
+(ns frontend.handler.file-sync
+  (:require ["path" :as path]
+            [cljs-time.coerce :as tc]
+            [cljs.core.async :as async :refer [go <!]]
+            [clojure.string :as string]
+            [frontend.config :as config]
+            [frontend.db :as db]
+            [frontend.fs.sync :as sync]
+            [frontend.handler.notification :as notification]
+            [frontend.state :as state]
+            [frontend.util :as util]
+            [frontend.util.persist-var :as persist-var]))
+
+(def hiding-login&file-sync (not config/dev?))
+(def refresh-file-sync-component (atom false))
+
+(defn graph-txid-exists?
+  []
+  (let [[graph-uuid _txid] @sync/graphs-txid]
+    (some? graph-uuid)))
+
+
+(defn create-graph
+  [name]
+  (go
+    (let [r* (<! (sync/create-graph sync/remoteapi name))
+          r (if (instance? ExceptionInfo r*) r* (:GraphUUID r*))]
+      (if (and (not (instance? ExceptionInfo r))
+               (string? r))
+        (do
+          (persist-var/-reset-value! sync/graphs-txid [r 0] (state/get-current-repo))
+          (persist-var/persist-save sync/graphs-txid)
+          (swap! refresh-file-sync-component not))
+        (if (= 404 (get-in (ex-data r) [:err :status]))
+          (notification/show! (str "create graph failed: already existed graph: " name) :warning)
```

Comment:
Could we capitalize all the notifications e.g. `Create` since this is user facing?